src/components/UsernameEntry/UsernameEntry.js

```diff
@@ -4,6 +4,21 @@ import './UsernameEntry.css';
 import ListSelection from '../listSelection/ListSelection';
 
 class UsernameEntry extends React.Component {
+  state = {
+    showConfirmButton: false,
+    selectedName: null
+  };
+
+  promptToConfirm = e => {
```

Comment:
Does the selected user object get passed here? Would `user` be a good name? I thought this was a mouse event initially